api/lib/notifications/notifications.js

```diff
@@ -69,9 +69,9 @@ function getMessageTitle(networks, notification) {
 
       return `Voting power decreased from ${Number(
         data.prevValidator.votingPower
-      ).toFixed(6)} to ${Number(data.nextValidator.votingPower).toFixed(
-        6
-      )} (${percentageDifference.toFixed(3)}% decrease) for ${
+      ).toFixed(2)}% to ${Number(data.nextValidator.votingPower).toFixed(
```

Comment:
refactor into some kind of helper function or at least the precision into a constant?

Reply:
OK, but then it makes sense to also change this one (voting difference) to precision 2 no? To have it as a meaningful proper constant: `(${percentageDifference.toFixed(3)}% increase)`

Reply:
yes exactly

Reply:
should be done in another PR